docs/content/4.sdk/2.getting-started/4.logger.md

```diff
@@ -0,0 +1,168 @@
+# Logger
+
+The Logger provides a logging utility for Alokai Storefront projects. It allows you to log messages at various levels and ability to provide additional context. 
+
+It is created to provide a unified way of logging messages across the project along with providing additional data out of the box.
+
+## Installation
+
+If the Logger is already included in the Alokai Storefront, you can skip this step.
+
+The Logger is provided by the SDK and framework specific packages.
+
+In order to install the Logger, you need to update following packages to at least the following versions:
+* `@vue-storefront/sdk` to `3.3.0`
+* `@vue-storefront/nuxt` to `6.2.0`
+* `@vue-storefront/next` to `4.3.0`
```

Comment:
Is it common to have migration guide in the getting started guide?
Storefronts does not have any migrations?

Reply:
We don't provide migration guides for storefronts, even tho those would be available internally.

Reply:
For this case, i wanted to specify a minimal version of packages when the logger becomes available.